### DISCUSSION TITLE
蘋果宣布iPod停產　庫存賣完正式走入歷史

### DISCUSSION THREAD

**Discussion by etin2468** [1101-GS4719-C/Python-Community-GS4719-C#648] [科技新知]
最近看阿哲的影片，ipod也終於要步入歷史了，在我小的時候還買過他的應該算最舊的款式，連螢幕都沒有的那種，只能說科技的進步已經讓許多的美好變成回憶了阿

https://news.tvbs.com.tw/tech/1789206

**Comment by bloodtrail0511** [comment]
好可惜喔~曾經陪我那麼久的系列說沒就沒了

**Comment by Jay-722** [comment]
難不成我前不久買的已經升級為永久最新版Ipad?

**Comment by Yuhsin000** [comment]
想買的時候我買不起，買得起的時候已經退居幕後了

**Comment by liangyinglly** [comment]
所以我小時候用的iPod現在留著是不是會增值

**Comment by stan890425** [comment]
想當年媽媽的車上有一台，現在被手機取代了

**Comment by JSHT** [comment]
iPod 在音樂走向數位化佔有重要地位。平常有在追蹤的 youtube 頻道 [电丸科技AK](https://www.youtube.com/c/AkilaZhang)，一個月前也介紹了 iPod 的各項產品 [802.11AK](https://youtu.be/sRuOvY6HBjs)，整體講述地很清晰。

**Comment by leon10456** [comment]
感謝樓主的分享 我的童年正式走入歷史QQ

**Comment by 110409538** [comment]
是童年的回憶,時代在更迭,很多事物都會成為歷史.
